src/utils/propsGenerator.js

```diff
@@ -50,5 +57,6 @@ export default function propsGenerator(
         transition,
       },
     },
+    ref,
```

Comment:
`{
   ... refCallback: { ref : (r) => { refCallback(r) : null }
}`

to avoid the let variable?

Reply:
yeh had that initially, then used the let variable to attempt to overcome the flow issue (which didn't work).  So can revert back to this